learn/experimental/reduce-indexing-memory-usage.mdx

```diff
@@ -0,0 +1,13 @@
+# Reduce indexing memory usage
+
+The `--experimental-reduce-indexing-memory-usage` configuration option enables `MDB_WRITEMAP`, an LLMDB option. Activating it can help reducing RAM usage in some UNIX and UNIX-like setups. However, it may also negatively impact write speeds and overall performance.
```

Comment:
```suggestion
The `--experimental-reduce-indexing-memory-usage` configuration option enables [`MDB_WRITEMAP`](https://github.com/LMDB/lmdb/blob/3947014aed7ffe39a79991fa7fb5b234da47ad1a/libraries/liblmdb/lmdb.h#L573-L581), an LLMDB option. Activating it can help reducing RAM usage in some UNIX and UNIX-like setups. However, it may also negatively impact write speeds and overall performance.
```
This link to the LMDB repo is available in the product discussion but might be useful here too

Reply:
Hmmm…that's an interesting idea, but after some consideration I prefer giving users a single way out of this page: the product discussion. I think linking to raw code in a repo not maintained by Meilisearch can be disorienting to new(ish) users who just want to improve their instance's performance. Especially when our docs page about this does not provide much help beyond this link.